### PR TITLE
Bug fixes/assigned ts

### PR DESCRIPTION
**Pull request by rripken**
Fixing the TimeSeries Groups so that they include the assigned time series.

### CODE REVIEW

File: cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesGroupDao.java [hunk 1]
```diff
@@ -27,70 +37,141 @@ public TimeSeriesGroupDao(DSLContext dsl)
...
-		Select select = step;
+		final RecordMapper<Record, Pair<TimeSeriesGroup, AssignedTimeSeries>> mapper = record15 -> {
+			TimeSeriesGroup group = buildTimeSeriesGroup(record15);
+			AssignedTimeSeries loc = buildAssignedTimeSeries(record15);
 
-		if ( officeId != null && !officeId.isEmpty())
+			return new Pair<>(group, loc);
+		};
+
+		SelectOnConditionStep<Record16<String, String, String, String, String, String, String, String, String, String, String, String, BigDecimal, BigDecimal, String, String>>
```
Comment (MikeNeilson) [inline cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesGroupDao.java]: FYI, if you're always using the named based getters later you can just do something like `SelectOnConditionStep<?>`

Not a huge deal but I imagine you like typing these out as much as I do.

File: cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesGroupDao.java [hunk 2]
```diff
@@ -27,70 +37,141 @@ public TimeSeriesGroupDao(DSLContext dsl)
...
+		{
+			List<AssignedTimeSeries> assigned = entry.getValue();
+
+			retval.add(new TimeSeriesGroup(entry.getKey(), assigned));
+		}
+		return retval;
 	}
 
-	public List<TimeSeriesGroup> getTimeSeriesGroups(String officeId, String categoryId, String groupId)
+	private AssignedTimeSeries buildAssignedTimeSeries(Record record15)
```
Comment (MikeNeilson) [inline cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesGroupDao.java]: This variable should probably have a better name than record15.

**Review by MikeNeilson — CHANGES_REQUESTED** [review]
It's looks good but while reading I didn't notice anything obvious about what the meaning of "record15" was... though it is 630 in the morning when I'm reading it.

File: cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesGroupDao.java (hunk 1 above)
Comment (rripken) [reply cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesGroupDao.java]: That makes sense.

File: cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesGroupDao.java (hunk 2 above)
Comment (rripken) [reply cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesGroupDao.java]: Agreed.

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**